lib/factory_bot/syntax/methods.rb

```diff
@@ -30,7 +30,7 @@ module Syntax
     ##   # factory with traits and attribute override
     ##   build_stubbed_list(:user, 15, :admin, :male, name: "John Doe")
     module Methods
-      # @!parse FactoryBot.register_default_strategies
+      # @!parse FactoryBot::Internal.register_default_strategies
```

Comment:
My yard doc knowledge is limited. What does `@!parse` do?

Reply:
It says that it parses the code as if it were present in that location. I think this was added since that happens explicitly to have it as part of the declaration of the method and to have yard doc add it as part of the class.